web/models/member.ex

```diff
@@ -21,11 +34,28 @@ defmodule Community.Member do
   def changeset(model, params \\ %{}) do
     model
     |> cast(params, @allowed_fields)
-    |> validate_required(:name)
-    |> validate_required(:email)
-    |> validate_required(:company_name)
+    |> validate_required([:name, :email, :company_name])
+    |> validate_url_format(:website)
+    |> validate_at_least_one_present([:website, :twitter_handle, :dribbble_username], "you must provide at least one social media contact")
+  end
+
+  defp validate_at_least_one_present(changeset, fields, error) do
```

Comment:
IMO If it's only used once then I think keeping it here makes sense. If you put it somewhere else it implies that it's used in two places, but it isn't. I don't feel strongly either way though